_components/sidenav.md

```diff
@@ -38,6 +38,7 @@ lead: "Hierarchical, vertical navigation to place at the side of a page."
       <li>Indicate where a user is within the navigational hierarchy. Use the “active” state to show users which page they have navigated to.</li>
       <li>Keep the navigation links short. They can be shorter derivatives of page titles themselves.</li>
       <li>If the navigation hierarchy is too long, users may miss items at the bottom. If it’s too deep, users may miss items that require too many clicks. Usability test to find the right balance between breadth and depth.</li>
+      <li>The width of the current menu item’s highlight border is set with the <code>$sidenav-current-border-width</code> variable.</li>
```

Comment:
@thisisdano technical or developer info is put in the "Implementation" section. See this as an example: https://standards.usa.gov/components/buttons/